dali/kernels/tensor_view.h

```diff
@@ -414,6 +414,33 @@ TensorListView<StorageGPU, T, ndim> make_tensor_list_gpu(T *data, TensorListShap
   return { data, std::move(shape) };
 }
 
+
+/**
```

Comment:
The whole header uses /// doxygen style. Please be consistent.

Reply:
This header is inconsistent with the rest of our code base. I want to be consistent with code base

Reply:
I think that file-scope consistency is more important than global one. Also, almost everything in dali/kernels uses this style.